gitium/functions.php

```diff
@@ -206,10 +206,50 @@ function gitium_commit_and_push_gitignore_file( $path = '' ) {
 	gitium_merge_and_push( $commit );
 }
 
+if ( ! function_exists( 'gitium_before_merge_and_push' ) ) :
+	function gitium_before_merge_and_push() {
+		global $gitium_lock_path, $gitium_lock_handle;
+
+		$gitium_lock_path   = apply_filters( 'gitium_lock_path', ABSPATH . 'wp-content/.gitium-lock' );
```

Comment:
The point of having a global `gitium_lock_path` is because we have two separate functions and in the second one we must use it. We can also drop the filter and we hard-coded the `gitium_lock_path` in both functions so we don't have to globalize it.